bd_spot_wrapper/spot_wrapper/april_tag_pose_estimator.py

```diff
@@ -38,9 +38,7 @@ class AprilTagPoseEstimator:
         image, mn_camera_T_marker = atpo.detect_markers_and_estimate_pose(image, should_render=True)
     """
 
-    def __init__(
-        self, spot: Spot, camera_intrinsics: dict, marker_length: float = MARKER_LENGTH
-    ):
+    def __init__(self, camera_intrinsics: dict, marker_length: float = MARKER_LENGTH):
```

Comment:
docstrings for april-tag pose estimator would be nice too.

Reply:
added